inst/pages/30_differential_abundance.qmd

```diff
@@ -568,6 +568,32 @@ zicoseq_res %>%
   knitr::kable()
 ```
 
+## Visualizing DAA
+
+In order to visualize results in a differential abundance analysis, we can use the `plotExpression` function from the scater library. This will show a jitterplot with the given results. Here's an example :
+
+```{r visualize DAA}
+library(scater)
+
+#Let's store the top three taxa
+
+top_taxa <- ancombc2_out$res %>%
+  dplyr::select(starts_with("taxon")) %>%
+  head(3) %>%
+  unlist()
```

Comment:
You can check output of ancombc2. It does also sensitivity test. --> You could also subset based on whether taxa passes the test. Something like this

`ind <- ancombc2_out$res[["passed_ss_patient_statusControl"]] & ancombc2_out$res[["diff_patient_statusControl"]]`